src/content/blog/kai-deep-dive-2024/index.mdx

```diff
@@ -0,0 +1,171 @@
+---
+author: jwmatthews
+date: 2024-05-07T07:00:00Z
+description: >-
+  Konveyor AI (Kai) Introduction
+featured: true
+image: header.png
+tags:
+  - Konveyor
+  - Kai
+  - GenAI
+  - AppModernization
+title: Kai - Generative AI Applied to Application Modernization
+---
+
+**Author**: [John Matthews](https://github.com/jwmatthews)
+
+### Overview
+
+Konveyor AI or "Kai", is an early effort of Generative AI applied to Application Modernization being explored under the Konveyor Ecosystem at [konveyor-ecosystem/kai](https://github.com/konveyor-ecosystem/kai).
+
+Kai implements a [Retrieval Augmented Generation (RAG)](https://www.promptingguide.ai/techniques/rag) approach that leverages data from Konveyor to help generate code suggestions to aid migrating legacy code bases to a different technology. The intent of this RAG approach is to shape the code suggestions to be similar to how an organization has solved problems in the past, without additional fine-tuning of the model.
+
+#### Demonstration Video
+
+The team has explored a use case of a Java EE application migrating to Quarkus for it's first example. Kai can be applied to other domains beyond Java EE -> Quarkus, the only requirement is that [analyzer-lsp](https://github.com/konveyor/analyzer-lsp) supports the language and there are sufficient [rulesets](https://github.com/konveyor/rulesets/tree/main/default/generated) defined for the target.
+
+![Kai Demo Loop](Kai_April_26c.gif)
+
+#### Below is a high level view of where Kai fits into a typical large scale modernization engagement.
+
+![Kai Konveyor Overview 2](Kai_Konveyor_Overview2.png)
+
+### What does Kai provide?
```

Comment:
Since this section is primarily covering the IDE integration, a header that directly mentions the IDE integration might make more sense